test/fixtures.py

```diff
@@ -19,29 +17,12 @@
 from kafka.client_async import KafkaClient
 from kafka.protocol.admin import CreateTopicsRequest
 from kafka.protocol.metadata import MetadataRequest
+from test.testutil import env_kafka_version, random_string
 from test.service import ExternalService, SpawnedService
 
 log = logging.getLogger(__name__)
 
 
-def random_string(length):
-    return "".join(random.choice(string.ascii_letters) for i in range(length))
```

Comment:
I had to move this out of this file, at least for now, to solve a circular import problem. That wasn't present in the original version of working on this, likely because I'd already deleted a ton of code related to `Simple*` code paths... so it can be moved back later if semantically better here. But for now simplest to move it out to solve the import problem.